ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -56,13 +56,14 @@ internal fun Footer(
     templateConfiguration: TemplateConfiguration,
     viewModel: PaywallViewModel,
     childModifier: Modifier = Modifier,
+    colors: TemplateConfiguration.Colors = templateConfiguration.getCurrentColors(),
     allPlansTapped: (() -> Unit)? = null,
 ) {
     Column {
         Footer(
             mode = templateConfiguration.mode,
             configuration = templateConfiguration.configuration,
-            colors = templateConfiguration.getCurrentColors(),
+            colors = colors,
```

Comment:
Footer colors depends on current tier colors for new template need to manually pass in the current tier colors

Reply:
I wonder if we need to make the transition between colors when switching tiers smoother... Feels slightly jumpy now... But maybe we should check with design to make sure.